frontend/src/components/CurrentSongView.tsx

```diff
@@ -54,6 +56,11 @@ function CurrentSongView() {
               <ArrowBackIcon />
             </Button>
           </Flex>
+          <div style={{ background: "white", padding: "16px" }}>
+            <a href={window.location.origin + "/addSong"} target="_blank">
```

Comment:
Probably a good idea to store the href value as a single variable so we're not generating the link twice